### ISSUE TITLE
differences to yogiben:admin

### ISSUE THREAD

**Issue by alexedev** [kaoskeya/meteor-admin#14]
Hi, @kaoskeya 

Could you please elaborate on why you decided to create a new package?
Was it created from scratch or based on `yogiben:admin`? 
If `yogiben:admin` was the starting point what were the main changes made? 

Thank you in advance.

ps I also forked `yogiben:admin` and changed the design, but then found your package and thought If it was better to use for my project. Your answers would be helpful for me to decide.

**Comment by kaoskeya** [comment]
Hi @buduguru,

I created this pacakge cos:
1. Getting my clients to use a different interface was a bit of a challenge and they preferred something that would have the same design throughout.
2. The pagination and a few others things were a little off and my reluctance to pick up coffeescript got in the way of fixing things.
3. I thought the whole admin panel could be implemented in a lot more simpler way and I prefer the dropdown switcher in place.

It was created from scratch. I only use the same config file format as yogiben:admin so that people can switch to this easily in case yogiben:admin does not work for them and also the other way around in case this package is breaking something. As an after thought I did feel some of the config stuff could have been different, but I think I'll stick it out for the ease of migration.

Hope this answers your question, feel free to comment if you have any more queries.

Thank you,
